docs/getting_started.md

```diff
@@ -1,199 +1,102 @@
-            1:
-            2:
-
-The `plot-*.py` scripts generate graphs. Open the PDF file that was created to see the graphed results.
-
-You can also parse and plot the TGen output using the `tgentools` program from the TGen repo. [This page describes how to get started](https://github.com/shadow/tgen/blob/main/doc/Tools-Setup.md).
-
-## Example experiment
-
-Consider a set of experiments where we would like to analyze the effect of changing our nodes' traffic queueing disciplines. We run the following 2 experiments:
-
-```bash
-cd resource/examples/
-rm -rf shadow.data shadow.log qdisc-fifo.data qdisc-fifo.log qdisc-rr.data qdisc-rr.log
-shadow --interface-qdisc fifo --data-directory qdisc-fifo.data shadow.config.xml > qdisc-fifo.log
-shadow --interface-qdisc rr   --data-directory qdisc-rr.data   shadow.config.xml > qdisc-rr.log
-```
+### Simulation Output
 
-To parse these log files, we use the following scripts:
+Shadow will write simulation output to the `shadow.data` directory. Each host has its own directory under `shadow.data/hosts`. For example:
 
 ```bash
-python3 ../../src/tools/parse-shadow.py --prefix=qdisc-fifo.results qdisc-fifo.log
-python3 ../../src/tools/parse-shadow.py --prefix=qdisc-rr.results   qdisc-rr.log
+$ ls -l shadow.data/hosts
+drwxrwxr-x 2 user user 4096 Jun  2 16:54 client1
+drwxrwxr-x 2 user user 4096 Jun  2 16:54 client2
+drwxrwxr-x 2 user user 4096 Jun  2 16:54 client3
+drwxrwxr-x 2 user user 4096 Jun  2 16:54 server
 ```
 
-Each of the directories `qdisc-fifo.results/` and `qdisc-rr.results/` now contain data statistics files extracted from the log files. We can now combine and visualize these results with the `plot-shadow.py` script:
+Each host directory contains the output for each process running on that host. For example:
 
 ```bash
-python3 ../../src/tools/plot-shadow.py --prefix "qdisc" --data qdisc-fifo.results/ "fifo" --data qdisc-rr.results/ "round-robin"
+$ ls -l shadow.data/hosts/client1
+-rw-rw-r-- 1 user user   1 Jun  2 16:54 client1.curl.1000.exitcode
+-rw-rw-r-- 1 user user   0 Jun  2 16:54 client1.curl.1000.shimlog
+-rw-r--r-- 1 user user   0 Jun  2 16:54 client1.curl.1000.stderr
+-rw-r--r-- 1 user user 542 Jun  2 16:54 client1.curl.1000.stdout
+
+$ cat shadow.data/hosts/client1/client1.curl.1000.stdout
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Directory listing for /</title>
+</head>
+<body>
+<h1>Directory listing for /</h1>
+...
 ```
 
-Then open the PDF file that was created to compare results from the experiments.
+Each host directory is also the [working directory](https://en.wikipedia.org/wiki/Working_directory) for the host's processes, which is why we specified `../../../server.py` as the path to the Python script in our Shadow configuration file (`./shadow.data/hosts/server/../../../server.py` → `./server.py`).
```

Comment:
No action needed for this PR, but I wonder if we should consider some shortcut for "shadow's working directory" and/or "the directory of the shadow config file". e.g. `$CONFIGDIR/server.py`

Reply:
Is this a tag we would look for in Shadow when parsing the config file?